mesonbuild/backend/backends.py

```diff
@@ -1354,6 +1362,27 @@ def create_install_data_files(self):
         with open(install_data_file, 'wb') as ofile:
             pickle.dump(self.create_install_data(), ofile)
 
+    def guess_install_tag(self, fname, outdir = None):
+        prefix = self.environment.get_prefix()
+        bindir = Path(prefix, self.environment.get_bindir())
+        libdir = Path(prefix, self.environment.get_libdir())
+        incdir = Path(prefix, self.environment.get_includedir())
+        localedir = Path(prefix, self.environment.coredata.get_option(mesonlib.OptionKey('localedir')))
+        dest_path = Path(prefix, outdir, Path(fname).name) if outdir else Path(prefix, fname)
+        if bindir in dest_path.parents:
```

Comment:
I think this should be:
```python
if dest_path.is_relative_to(bindir):
```

Reply:
You live too far into the future: `New in version 3.9.` ;-)